src/LP/LP.jl

```diff
@@ -163,17 +233,26 @@ function write_objective(io::IO, model::Model, sanitized_names::Dict{MOI.Variabl
 end
 
 function MOI.write_to_file(model::Model, io::IO)
+    # Reset the global variables about warnings.
+    global warned_start
+    warned_start = Set{Char}()
+    global warned_illegal
+    warned_illegal = Set{Char}()
```

Comment:
Instead of using global variables, just pass these around as function arguments.

Reply:
I added these to `Options`, otherwise I would have to add many new arguments on all function calls. Or would you prefer another solution?